### ISSUE TITLE
chinese support?

### ISSUE THREAD

**Issue by Lukangkang123** [Muennighoff/sgpt#25]
Does the model support Chinese input?

**Comment by Muennighoff** [comment]
Sure; For asymmetric search (e.g. retrieval), I'd recommend https://huggingface.co/bigscience/sgpt-bloom-7b1-msmarco which has seen lots of Chinese during pretraining

**Comment by Lukangkang123** [comment]
> Sure; For asymmetric search (e.g. retrieval), I'd recommend https://huggingface.co/bigscience/sgpt-bloom-7b1-msmarco which has seen lots of Chinese during pretraining

Thanks you very much! Do you mean this code?:[https://github.com/Muennighoff/sgpt#asymmetric-semantic-search-be](https://github.com/Muennighoff/sgpt#asymmetric-semantic-search-be)

**Comment by Muennighoff** [comment]
> > Sure; For asymmetric search (e.g. retrieval), I'd recommend https://huggingface.co/bigscience/sgpt-bloom-7b1-msmarco which has seen lots of Chinese during pretraining
> 
> Thanks you very much! Do you mean this code?:https://github.com/Muennighoff/sgpt#asymmetric-semantic-search-be

Yeah you can use that code and swap the model for [bigscience/sgpt-bloom-7b1-msmarco](https://huggingface.co/bigscience/sgpt-bloom-7b1-msmarco)

**Comment by Lukangkang123** [comment]
> > > Sure; For asymmetric search (e.g. retrieval), I'd recommend https://huggingface.co/bigscience/sgpt-bloom-7b1-msmarco which has seen lots of Chinese during pretraining
> > 
> > 
> > Thanks you very much! Do you mean this code?:https://github.com/Muennighoff/sgpt#asymmetric-semantic-search-be
> 
> Yeah you can use that code and swap the model for [bigscience/sgpt-bloom-7b1-msmarco](https://huggingface.co/bigscience/sgpt-bloom-7b1-msmarco)


> > > Sure; For asymmetric search (e.g. retrieval), I'd recommend https://huggingface.co/bigscience/sgpt-bloom-7b1-msmarco which has seen lots of Chinese during pretraining
> > 
> > 
> > Thanks you very much! Do you mean this code?:https://github.com/Muennighoff/sgpt#asymmetric-semantic-search-be
> 
> Yeah you can use that code and swap the model for [bigscience/sgpt-bloom-7b1-msmarco](https://huggingface.co/bigscience/sgpt-bloom-7b1-msmarco)

“swap the model for [bigscience/sgpt-bloom-7b1-msmarco](https://huggingface.co/bigscience/sgpt-bloom-7b1-msmarco)”? Do you mean to replace the two lines of code 
`tokenizer = AutoTokenizer.from_pretrained("Muennighoff/SGPT-125M-weightedmean-msmarco-specb-bitfit")
model = AutoModel.from_pretrained("Muennighoff/SGPT-125M-weightedmean-msmarco-specb-bitfit")`
with 
`tokenizer = AutoTokenizer.from_pretrained("Muennighoff/sgpt-bloom-7b1-msmarco")
model = AutoModel.from_pretrained("Muennighoff/sgpt-bloom-7b1-msmarco)`？
 I tried to do this, but was prompted that the model ‘sgpt-bloom-7b1-msmarco’ does not exist.

Can I use the code from [here](https://github.com/Muennighoff/sgpt#symmetric-semantic-search-be) directly? Or does it need to be modified accordingly?

**Comment by Lukangkang123** [comment]
> > > Sure; For asymmetric search (e.g. retrieval), I'd recommend https://huggingface.co/bigscience/sgpt-bloom-7b1-msmarco which has seen lots of Chinese during pretraining
> > 
> > 
> > Thanks you very much! Do you mean this code?:https://github.com/Muennighoff/sgpt#asymmetric-semantic-search-be
> 
> Yeah you can use that code and swap the model for [bigscience/sgpt-bloom-7b1-msmarco](https://huggingface.co/bigscience/sgpt-bloom-7b1-msmarco)

Thanks you!
